keyboards/pimoroni/keybow2040/keybow2040.c

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+#include "rgb_matrix.h"
+
+void keyboard_post_init_user(void) {
+	debug_enable = true;
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+#ifdef CONSOLE_ENABLE
+	dprintf("KL: kc: 0x%04X, col: %2u, row: %2u, "
+		"pressed: %u, time: %5u, "
+		"int: %u, count: %u\n",
+		keycode, record->event.key.col, record->event.key.row,
+		record->event.pressed, record->event.time,
+		record->tap.interrupted, record->tap.count);
+#endif
+  return true;
+}
+
+// https://github.com/adafruit/Adafruit_CircuitPython_IS31FL3731/blob/main/adafruit_is31fl3731/keybow2040.py
+
+const is31_led PROGMEM g_is31_leds[RGB_MATRIX_LED_COUNT] = {
+	{ 0, 120, 88, 104 }, // 0, 0
+	{ 0, 136, 40, 72 },  // 1, 0
+	{ 0, 112, 80, 96 },  // 2, 0
+	{ 0, 128, 32, 64 },  // 3, 0
+	{ 0, 121, 89, 105 }, // 0, 1
+	{ 0, 137, 41, 73 },  // 1, 1
+	{ 0, 113, 81, 97 },  // 2, 1
+	{ 0, 129, 33, 65 },  // 3, 1
+	{ 0, 122, 90, 106 }, // 0, 2
+	{ 0, 138, 25, 74 },  // 1, 2
+	{ 0, 114, 82, 98 },  // 2, 2
+	{ 0, 130, 17, 66 },  // 3, 2
+	{ 0, 123, 91, 107 }, // 0, 3
+	{ 0, 139, 26, 75 },  // 1, 3
+	{ 0, 115, 83, 99 },  // 2, 3
+	{ 0, 131, 18, 67 },  // 3, 3
+};
```

Comment:
These numbers look wrong for the IS31FL3731 driver in QMK — that driver expects the register addresses which match this part of the IS31FL3731 datasheet (the PWM Register columns):
![IS31FL3731 datasheet](https://github.com/qmk/qmk_firmware/assets/616082/ff4a4a7f-ad6a-4922-92b9-2a54e25f9ede)

So the range of register addresses should be `0x24`…`0xB3` (36…179).  I guess that CircuitPython number the PWM registers starting from 0, so you need to add `0x24` to these numbers.  The next step would be replacing those raw numbers with the matching defines from [is31fl3731.h](https://github.com/qmk/qmk_firmware/blob/aa91d98531a299f86f9dab8875dcf46983b16af1/drivers/led/issi/is31fl3731.h#L50-L210) — those defines match the numbering in the datasheet.

Reply:
Excellent, thanks for spotting that mistake! I was too lazy to replace the raw numbers with the register names: regrettable, because I would have noticed the mismatch if I had cleaned them up.

Reply:
This is fixed in round two of the PR.